### PR TITLE
Prevent interrupted exception from polluting log during supervisor shutdown

### PR DESCRIPTION
**Pull request by pjain1**

### CODE REVIEW

File: extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisor.java [hunk 1]
```diff
@@ -1794,6 +1794,9 @@ public Long apply(Integer key, Long existingOffsetInMap)
               ServiceMetricEvent.builder().setDimension("dataSource", dataSource).build("ingest/kafka/lag", lag)
           );
         }
+        catch (InterruptedException e) {
+          log.warn("Interrupted while computing Kafka lag");
```
Comment (gianm) [inline extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisor.java]: Just doing nothing would be fine, I think. I can't see a need for a log message. Up to you though @pjain1, I'm fine with it either way.

Comment (pjain1) [inline extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisor.java]: ok..changed it to no-op

**Comment by gianm** [comment]
@pjain1 was this patch no good?

**Comment by pjain1** [comment]
delete branch by mistake, reopened the PR

**Pull request merged**